addon/components/o-s-s/progress-bar.ts

```diff
@@ -0,0 +1,50 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+enum ProgressBarSkins {
+  'primary',
+  'warning',
+  'success'
+}
```

Comment:
With an `enum` here and not type definition like `type ProgressBarSkins = 'primary' | 'warning' | 'success'`